### DISCUSSION TITLE
Detailed debug messages

### DISCUSSION THREAD

**Discussion by tomkom** [quarto-dev/quarto-cli#4720] [Q&A]
Is there a way to get detialed debug messages (lines where error happens) from the render process?
I have a project that renders (based on two separate yml file, I would love  to keep it in one but not sure how), to revealjs and to a book (as html and pdf). Mermaid diagrams caused me issues with the pdf pipeline, so I now have to have alternate images, included based on 

```
::: {.content-visible when-format="revealjs"}
::: {mermaid}
figure here
:::
::: {.content-visible when-format="pdf"}
png figure here
:::
```

Now, sometimes I mess up, and while slides generate fine, I get errors from the html/pdf pipeline. 

```
test.qmd
ERROR: Internal error: temporarily disabled until deno 1.28.* gets puppeteer support
```

I know this should be resolved sometimes soon, but in any way, this is not a very informative message. I tried with --verbose, but there is no much there. Is there a --debug (does not seem) or similar mode where a failure could poitn to the exact line where the problem was encountered?

Thanks,
Martin

**Comment by cscheid** [comment]
This is now resolved on 1.3 prereleases.

**Reply by tomkom** [reply]
How? I am using 1.3.217 and we had this discussion already in another thread (https://github.com/quarto-dev/quarto-cli/discussions/4441). The main gist is however the debug message. Is there a way to get debugs? 
This issue thread here is not about the specific problem of mermaid, apologies if this was not clear. I will also add a note into the original thread on mermaid after testing with a new pre-release (do I need to install chromium extension?_.

**Reply by cscheid** [reply]
You can set `QUARTO_PRINT_STACK=true`, but I don't think that's going to be much help here because we know the source of the problem; it's somehow the instance of the chromium executable that is being detected by Quarto which doesn't expose the necessary entry points.

You mention 1.3.217, but our latest release as of the time of my message is 1.3.253.

**Reply by tomkom** [reply]
I have now updated to 1.2.335 (latest I can see) and chromium quits unexpectedly, and the whole thing still fails. This is on Mac Vnetura on M1.

tools:
```

Tool         Status            Installed     Latest
chromium     Up to date        869685        869685
```

**Reply by cderv** [reply]
> I have now updated to 1.2.335 (latest I can see)

This is the latest stable release - Here is direct link to prerelease download page https://quarto.org/docs/download/prerelease.html but it can be found also on main download page in right tab https://quarto.org/docs/download/
